src/server/pps/server/api_server.go

```diff
@@ -2351,7 +2351,10 @@ func (a *apiServer) CreateDetPipelineSideEffects(ctx context.Context, pipeline *
 		},
 	}
 	s.SetLabels(map[string]string{
-		"suite": "pachyderm",
+		"suite":        "pachyderm",
+		"pipelineName": pipeline.Name,
+		"project":      pipeline.GetProject().String(),
+		"determined":   "true",
```

Comment:
Rather than `determined=true`, I wonder if this should be `type=determined` or similar.

Reply:
great suggestion <3. I'll open a ticket for this and handle it in a separate PR